dtls/tutorial-udp-dtls.md

```diff
@@ -164,7 +164,7 @@ We can then call SERV_PORT where it is needed and if you, the client, are alread
     4.4 “ntohl”
         network to host - long : convert a 32-bit number from a network representation into the local processor`s format. This is commonly used to read an IP address from a sockaddr structure.
 
-        Using any of the above 4.4 macros will guarantee that your code remains portable regardless 
+        Using any of the above 4.4 macros will guarantee that your code remains portable regardless
         of the architecture you use in compilation.
 
 ####1.5. <BEGIN LOOP>:
```

Comment:
I don't think this markdown will show up correctly.  Please take another look at this README.md on GitHub, from your fork.  The headings should be bold.  This specific location needs a space after the "####".